docs/Scripting/PublicScripts.md

```diff
@@ -0,0 +1,49 @@
+---
+id: PublicScripts
+title: Public Scripts
+---
+
+
+Enklu Cloud comes with an ever-expanding library of public scripts. These scripts are available to all Enklu Cloud accounts and may be used in any experience.
+
+## Script Library
+
+The script library is located in the panel below the canvas. On the left side of the script library is a list of all the experiences your account has access to, meaning you can access scripts across multiple expereinces. There is also a **Public** category near the top of this list.  It gives you access to of all of the currently available public scripts.
```

Comment:
remove the first "of" in the last sentence (so "it gives you access to all of the currently available public scripts")